lib/fileutils.rb

```diff
@@ -817,7 +817,8 @@ def cp(src, dest, preserve: nil, noop: nil, verbose: nil)
   alias copy cp
   module_function :copy
 
-  # Recursively copies files from +src+ to +dest+.
+  # Recursively copies files from +src+ (a single path or an array of paths)
+  # to +dest+ (a single path).
   #
   # Arguments +src+ and +dest+
```

Comment:
For consistency, I think we should specify here that `src` is a single or an array of paths and `dest` is a single path.

Reply:
The docs for this method says:

> Recursively copies files from +src+ (a single path or an array of paths)
> to +dest+ (a single path).
> 
> Arguments +src+ and +dest+
> should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].

Whereas, for example, the docs for `ln_s` says:

> Creates {symbolic links}[https://en.wikipedia.org/wiki/Symbolic_link].
>
> Arguments +src+ (a single path or an array of paths)
> and +dest+ (a single path)
> should be {interpretable as paths}[rdoc-ref:FileUtils@Path+Arguments].

So yeah, perhaps we can remove duplication in the docs.